File: spring-boot/login-form-thymeleaf/src/main/java/no/acntech/sandbox/controller/ViewController.java
package no.acntech.sandbox.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.servlet.ModelAndView;

@Controller
public class ViewController {

    @GetMapping(path = "/")
    public ModelAndView getIndexPage() {
        return new ModelAndView("index");
    }

    @GetMapping(path = "/about")
    public ModelAndView getAboutPage() {
        return new ModelAndView("about");
    }

    @GetMapping(path = "/admin")
    public ModelAndView getAdminPage() {
        return new ModelAndView("admin");
    }

    @GetMapping(path = "/login")
    public String getLoginPage() {
        return "login";
    }
}

File: spring-boot-kotlin/oauth2-client-login/src/main/kotlin/no/acntech/sandbox/controller/ViewController.kt
package no.acntech.sandbox.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class ViewController {

    @GetMapping(path = ["/"])
    fun getIndexPage(): String {
        return "index"
    }

    @GetMapping(path = ["/about"])
    fun getAboutPage(): String {
        return "about"
    }
}